problems/objects.js

```diff
@@ -187,15 +301,57 @@ const countOccurance = () => {};
  * @returns {Object} Counts all characters except spaces  {a: 2, g: 1, o: 2, d:1, s: 1, n:a, k:1, e: 1}
  */
 
-const countOccuranceNoSpaces = () => {};
+const countOccuranceNoSpaces = () => {
+  // let lowerStr = str.toLowerCase(); //how do I take out the spaces?
+  // let counter = {};
+  // for (let i = 0; i <= lowerStr.length - 1; i++) {
+  //   let strI = lowerStr[i];
+  //   if (counter[strI] === " ") {
+  //     delete counter[strI];
```

Comment:
so instead of deleting it, you want to just not add " " to the object

Reply:
got it now. I used .replace()